environments/icds-cas/proxy.yml

```diff
@@ -11,5 +11,8 @@ special_sites:
   - cas_ssl
   - cas_sentry
 
+special_sites:
```

Comment:
this is the same key as above so would override it

Reply:
changed var